indico/migrations/versions/20210813_1633_dc53d6e8c576_make_event_category_nullable.py

```diff
@@ -0,0 +1,23 @@
+"""Make event category nullable
+
+Revision ID: dc53d6e8c576
+Revises: 9d00917b2fa8
+Create Date: 2021-07-07 16:48:03.719901
+"""
+
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = 'dc53d6e8c576'
+down_revision = '9d00917b2fa8'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.drop_constraint('ck_events_category_data_set', 'events', schema='events')
+
+
+def downgrade():
+    op.create_check_constraint('category_data_set', 'events', 'category_id IS NOT NULL OR is_deleted', schema='events')
```

Comment:
Maybe we should also flag all events with no category that aren't deleted as deleted? Otherwise a downgrade fails (which is a bit annoying during development)

Reply:
Ah, good idea, I was deleting them manually but this is much more convenient.